src/main/java/com/glodblock/github/crossmod/waila/PartWailaDataProvider.java

```diff
@@ -49,7 +49,7 @@ public ItemStack getWailaStack(final IWailaDataAccessor accessor, final IWailaCo
             ItemStack wailaStack = null;
```

Comment:
Similar to [here](https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/133/commits/b77116b584f1682085293a913f2477f2ed9440c9), you can drop `null` assignment.

Reply:
Seems like it's not possible.
![image](https://github.com/GTNewHorizons/AE2FluidCraft-Rework/assets/73699379/3c570b7c-d977-4da9-83d1-677cf87fa47c)

Reply:
Rephrasing the [following](https://stackoverflow.com/questions/2448843/variable-might-not-have-been-initialized-error) issue,
> Their default value is given when declared as class members. Local variables don't have default values

Reply:
Oh, yeah that makes sense.